invenio_app_ils/ui/backoffice/src/routes/index.js

```diff
@@ -1,2 +1,3 @@
 export { ItemRoutes } from './ItemRoutes';
 export { LoanRoutes } from './LoanRoutes';
+export { UserRoutes } from './UserRoutes';
```

Comment:
Why we should have all these Route components? The `Switch` component here is redundant I think.

Reply:
Fixed by me in the next refactoring to merge everything in one app.